README.md

```diff
@@ -15,6 +17,36 @@ cd ../src
 jupyter notebook
 ```
 
+### Quick Start With NVIDIA AI Workbench
+
+[NVIDIA AI Workbench](https://www.nvidia.com/en-us/deep-learning-ai/solutions/data-science/workbench/) is a development environment manager that enables data scientists and developers to create, customize, collaborate, and migrate AI workloads and applications seamlessly across GPU systems, from laptops to cloud.
+
+If you do not NVIDIA AI Workbench installed, first complete the installation for AI Workbench [here](https://docs.nvidia.com/ai-workbench/user-guide/latest/installation/overview.html). 
```

Comment:
Just a typo, should read "If you do not **have** NVIDIA AI Workbench installed, ..."

Reply:
Good catch, will fix

Reply:
fixed in most recent commit